src/access/wac.ts

```diff
@@ -162,3 +187,26 @@ export async function getPublicAccess(
 ): Promise<Access | null> {
   return getActorClassAccess(resource, getPublicAccessWac, options);
 }
+
+/**
+ * For a given Resource, look up its metadata, and read the Access permissions
+ * granted explicitly to each individual Agent.
+ *
+ * Note that this only lists permissions granted to each Agent individually,
+ * and will not exhaustively list modes any Agent may have access to because
+ * they apply to everyone, or because they apply to an Agent through a group for
+ * instance.
+ *
+ * @param resource The URL of the Resource for which we want to list Agents Access
+ * @param options Optional parameter `options.fetch`: An alternative `fetch` function to make the HTTP request, compatible with the browser-native [fetch API](https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/fetch#parameters).
+ * @returns A map of Agents WebIDs associated to a list of modes: True for Access modes
+ * granted to the Agent, False for Access modes denied to the Agent, and undefined otherwise.
+ */
+export async function getAgentAccessAll(
```

Comment:
Since we're not awaiting things. (This might also be relevant to the functions you added earlier; I didn't check.)

```suggestion
export function getAgentAccessAll(
```

Reply:
Thanks for that, I didn't realize that could be an issue : see https://twitter.com/_jayphelps/status/1324565522755788803

Reply:
975cb80ffc8c80844480922d3dcf4919bad72ade